android/src/main/java/com/tailscale/ipn/mdm/MDMSettingsDefinitions.kt

```diff
@@ -4,14 +4,32 @@
 package com.tailscale.ipn.mdm
 
 import android.os.Bundle
+import android.util.Log
 import com.tailscale.ipn.App
 import com.tailscale.ipn.ui.util.set
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 
-abstract class MDMSetting<T>(defaultValue: T, val key: String, val localizedTitle: String) {
+// Don't keep track of the key if shouldRegisterKey is false - shouldRegisterKey should be false when creating an MDMSetting for retrieving a setting.
```

Comment:
Does this mean that all initializer calls in MDMSettings.kt need to be updated with `shouldRegisterKey: false`?. The default value here is `true`.

Reply:
no, they should register keys. when we are checking to see if there exists an MDMSetting for a certain key, shouldRegisterKey should be false. I updated the comment to hopefully be a little clearer